src/components/payment/payment.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import styles from './payment.module.scss';
+import commonStyles from '@styles/common-styles.module.scss';
+import { SingleInput } from '../single-input/single-input';
+
+export interface PaymentProps {
+    className?: string;
+}
+
+export const Payment = ({ className }: PaymentProps) => {
+    return (
+        <div className={classNames(styles.root, className, styles.paymentdiv)}>
+            <h3 className={styles.title}>Payment</h3>
+            <h2 className={styles.boldMessage}>
+                We&apos;re not able to accept online payments at the moment.
+            </h2>
+            <h3>Please contact us to complete your purchase. Thanks. </h3>
```

Comment:
you use heading elements (<h /> )everywhere. I don't think we really need it. Replace some of them with <span/> or whatever is semantically more appropriate.